app/src/main/java/com/example/parkingsystemjava/activity/ReservationActivity.java

```diff
@@ -0,0 +1,35 @@
+package com.example.parkingsystemjava.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import com.example.parkingsystemjava.databinding.ActivityReservationBinding;
+import com.example.parkingsystemjava.mvp.contract.ReservationActivityContract;
+import com.example.parkingsystemjava.mvp.presenter.ReservationPresenter;
+import com.example.parkingsystemjava.mvp.view.ReservationView;
+
+public class ReservationActivity extends AppCompatActivity {
+    private ActivityReservationBinding binding;
+    private ReservationActivityContract.PresenterContract presenter;
+
+    public static Intent newInstance(Context context) {
+        Intent intent = new Intent(context, ReservationActivity.class);
```

Comment:
create a variable is not necessary you can return this without creating the variable
`return new Intent(context, ReservationActivity.class);`

Reply:
Done, thanks!